packages/flutter_tools/test/general.shard/resident_runner_test.dart

```diff
@@ -1570,7 +1570,13 @@ flutter:
         flutterDevice,
       ],
       stayResident: false,
-      debuggingOptions: DebuggingOptions.enabled(BuildInfo.debug),
+      debuggingOptions: DebuggingOptions.enabled(
+        const BuildInfo(
+          BuildMode.debug,
+          '',
```

Comment:
Unless I'm missing something I think this should be `null`.
```suggestion
          null,
```

Reply:
Done